<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tpmsgenerator</class>
 <widget class="QDialog" name="tpmsgenerator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="randFiberGenerationsLabel">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 75 16pt &quot;Serif&quot;;</string>
           </property>
           <property name="text">
            <string>Triply Periodic Minimal Surface</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 11pt &quot;Serif&quot;;</string>
           </property>
           <property name="text">
            <string>Domain Dimensions</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="xSizeLabel">
             <property name="styleSheet">
              <string notr="true">font: 10pt &quot;Serif&quot;;</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::Panel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Sunken</enum>
             </property>
             <property name="text">
              <string>x-size:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="xSizeEdit">
             <property name="styleSheet">
              <string notr="true">font: 10pt &quot;Serif&quot;;</string>
             </property>
             <property name="text">
              <string>200</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="ySizeLabel">
             <property name="styleSheet">
              <string notr="true">font: 10pt &quot;Serif&quot;;</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::Panel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Sunken</enum>
             </property>
             <property name="text">
              <string>y-size:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="ySizeEdit">
             <property name="styleSheet">
              <string notr="true">font: 10pt &quot;Serif&quot;;</string>
             </property>
             <property name="text">
              <string>200</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="zSizeLabel">
             <property name="styleSheet">
              <string notr="true">font: 10pt &quot;Serif&quot;;</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::Panel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Sunken</enum>
             </property>
             <property name="text">
              <string>z-size:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="zSizeEdit">
             <property name="styleSheet">
              <string notr="true">font: 10pt &quot;Serif&quot;;</string>
             </property>
             <property name="text">
              <string>200</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="fibAngleVarLabel">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Serif&quot;;</string>
     </property>
     <property name="text">
      <string>Parameters</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="xminLabel">
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Serif&quot;;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string>w min: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="wMinEdit">
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Serif&quot;;</string>
       </property>
       <property name="text">
        <string>0.1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="xMaxLabel">
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Serif&quot;;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string>w max: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="wMaxEdit">
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Serif&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="qMinLabel">
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Serif&quot;;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string>q min: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="qMinEdit">
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Serif&quot;;</string>
       </property>
       <property name="text">
        <string>0.5</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="qMaxLabel">
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Serif&quot;;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string>q max: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="qMaxEdit">
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Serif&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>37</width>
         <height>17</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="Generate">
       <property name="minimumSize">
        <size>
         <width>175</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 11pt &quot;Serif&quot;;</string>
       </property>
       <property name="text">
        <string>Generate</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
